conductr_cli/sandbox_run_jvm.py

```diff
@@ -467,6 +468,11 @@ def download_sandbox_image(image_dir, package_name, artefact_type, image_version
     try:
         bintray_auth = bintray_resolver.load_bintray_credentials(raise_error=False)
 
+        if sandbox_version.is_conductr_on_private_bintray(image_version):
+            bintray_repo = BINTRAY_CONDUCTR_COMMERCIAL_REPO
+        else:
+            bintray_repo = BINTRAY_CONDUCTR_COMMERCIAL_REPO if bintray_auth[0] else BINTRAY_CONDUCTR_GENERIC_REPO
```

Comment:
@markusjura - I think this is necessary since if Bintray credential is present we want to direct user to download from `lightbend/commercial-releases` so we can track the download numbers.